### ISSUE TITLE
NTE: In game commands not working

### ISSUE THREAD

**Issue by Chalwk** [sgtcaze/NametagEdit#73]
When we type /nte (by itself) or /nte [argument], nothing actually happens in game. 
There is no command syntax error. 

The only commands that appear to actually work in game are /nte reload and /nte debug. 

NameTagEdit's 'function' seems to be be working fine; that is, nametags do actually appear above the players head and in the tablist. It's just the plugin commands that seem to be a problem.

**Using the latest dev build from jenkins.** 
 Build #35 (30-Jul-2016 04:10:34)

**This error pops up in the console when we type /nte:**

```
01.08 22:22:11 [Server] INFO at java.lang.Thread.run(Thread.java:745) [?:1.8.0_25]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:571) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:672) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:399) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:733) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_25]
01.08 22:22:11 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_25]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:1183) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at net.minecraft.server.v1_10_R1.PlayerConnection.handleCommand(PlayerConnection.java:1348) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at org.bukkit.craftbukkit.v1_10_R1.CraftServer.dispatchCommand(CraftServer.java:646) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) [spigot_1.10.jar:git-Spigot-6016ac7-10c10b3]
01.08 22:22:11 [Server] INFO at com.nametagedit.plugin.NametagCommand.onCommand(NametagCommand.java:34) [NametagEdit.jar:?]
01.08 22:22:11 [Server] INFO at com.nametagedit.plugin.NametagCommand.sendUsage(NametagCommand.java:66) [NametagEdit.jar:?]
01.08 22:22:11 [Server] INFO at mkremins.fanciful.FancyMessage.send(FancyMessage.java:682) [Freeze.jar:?]
01.08 22:22:11 [Server] INFO at mkremins.fanciful.FancyMessage.send(FancyMessage.java:617) [Freeze.jar:?]
01.08 22:22:11 [Server] INFO at mkremins.fanciful.FancyMessage.createChatPacket(FancyMessage.java:644) ~[Freeze.jar:?]
01.08 22:22:11 [Server] INFO at java.lang.Integer.parseInt(Integer.java:615) ~[?:1.8.0_25]
01.08 22:22:11 [Server] INFO at java.lang.Integer.parseInt(Integer.java:580) ~[?:1.8.0_25]
01.08 22:22:11 [Server] INFO at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65) ~[?:1.8.0_25]
01.08 22:22:11 [Server] INFO java.lang.NumberFormatException: For input string: "R"
01.08 22:22:11 [Server] WARN Argument could not be passed.
01.08 22:22:11 [Server] INFO RuEvenPro issued server command: /nte`
```

**Comment by sgtcaze** [comment]
Looks like the library we're using hasn't been updated properly. I will check that resource, and if we can't get clickables to work I will update it.

Edit: Dev build 36 uses a new version of the library http://ci.playmc.cc/job/NametagEdit/36/

**Comment by Chalwk** [comment]
Unfortunately the problem seems to persist with the same error.

**Comment by sgtcaze** [comment]
Alright I will update it @Chalwk77 and tag you when the commit is up/builds are available.

**Comment by Chalwk** [comment]
Thank you (:

**Comment by sgtcaze** [comment]
Hmm are you sure you're using Build 36+ on the Jenkins? I've been scratching my head, as fanciful looks updated. I tried using /nte in-game and in console with Spigot version 90f61bc-83a9dbd (1.10.2). I do not encounter the same error.

**Comment by Chalwk** [comment]
I'm afraid so. Perhaps there is a plugin conflict.
Shall I list my plugins?
- Console does not throw an error for me. Just in-game.

We're now on Jenkins Build #39, and the issue remains. I'll do a bit more investigating on my end and update if I find anything. 

Edit: **/nte groups list** works in-game now.

**Comment by sgtcaze** [comment]
You are using MC version 1.10 right? You can safely update to the same version (newest 1.10.2 revision) I am using without any NMS breaking.

**Comment by Chalwk** [comment]
Sorry for the unfashionably late reply, however it's bad news. 
I was indeed using MC v1.10 and updated to 1.10.2. The aforementioned issue remains.
http://pastebin.com/1CHhZAiC

Edit: **Disabled all plugins except for NTE, and now all commands work**. My community server runs 119 plugins. I'll take some time to disable them 1 by 1 until I have the culprit. Will update again soon.

**Comment by sgtcaze** [comment]
That is super strange! I am looking at that error wondering how it's possible. It is trying to insert an "R" in /ne command syntax. Do you have a plugin that interacts with NametagEdit to your knowledge?

**Comment by Chalwk** [comment]
Update: (good news)

I don't know how I missed this, but if you look at the error closly, it mentions "freeze.jar".
**08.08 23:46:44 [Server] INFO at mkremins.fanciful.FancyMessage.createChatPacket(FancyMessage.java:644) ~[Freeze.jar:?]**

I happen to use a plugin by that name. So I disabled it, and hard-restarted the server. All /nte commands are working properly in game now. Bizarre, right?

The plugin mentioned:
https://www.spigotmc.org/resources/afreeze.11582/

Some quick feedback on NTE in general:
- Love the plugin. Fantastic project. 
  10/10

~ Edit:
I have contacted the developer of **afreeze** and sent him a link to this report for reference. He replied and said the situation was very strange although his plugin is outdated apparently - he has intentions of updating it soon.
